docs/iris/src/whatsnew/latest.rst

```diff
@@ -81,7 +81,7 @@ This document explains the changes made to Iris for this release
 ===========
 
 #. `@rcomer`_ removed an old unused test file. (:pull:`3913`)
-
+#. `@jamesp`_ updated a test to the latest numpy version (:pull: `3977`)
```

Comment:
@jamesp Could you add an extra blank line above i.e., between lines 83-84 :+1: 

Also, since you're a new `iris` core dev (:partying_face:) could you add a reference to yourself in the `docs/iris/src/common_links.inc` under the `Core developers (@github names) in alphabetical order` section.

This is a one time thing. It's a convenience to core devs in the documentation so they don't have to keep making repeated `rst` links to reference their GitHub handle in multiple places.

Reply:
Hopefully should be all done now :smile:

Reply:
@jamesp Ahhh, sorry...

Could you remove the space after the `:`
```rst
(:pull: `3977`)
```
to
```rst
(:pull:`3977`)
```
as the `rst` isn't rendering the GitHub pull-request directive correctly:
<img width="384" alt="rtd-jamesp" src="https://user-images.githubusercontent.com/2051656/106759119-42c81300-662a-11eb-8e62-ce5b6c96e973.PNG">

Reply:
done!